src/scr.c

```diff
@@ -916,24 +993,33 @@ static int scr_get_params()
               __FILE__, __LINE__
       );
     }
-  }
+  } 
```

Comment:
Let's drop the trailing white space on this line
```suggestion
  }
```